presto-main/src/main/java/com/facebook/presto/sql/planner/plan/IndexSourceNode.java

```diff
@@ -67,6 +67,11 @@ public IndexSourceNode(
         checkArgument(!outputSymbols.isEmpty(), "outputSymbols is empty");
         checkArgument(assignments.keySet().containsAll(lookupSymbols), "Assignments do not include all lookup symbols");
         checkArgument(outputSymbols.containsAll(lookupSymbols), "Lookup symbols need to be part of the output symbols");
+        Set<ColumnHandle> assignedColumnHandles = ImmutableSet.copyOf(assignments.values());
```

Comment:
I had to go look up how to do this.  This page seemed to be a good resource:  https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History